package-lock.json

```diff
@@ -261,7 +261,7 @@
     },
     "node_modules/@aws-crypto/crc32": {
       "version": "3.0.0",
-      "resolved": "https://registry.npmjs.org/@aws-crypto/crc32/-/crc32-3.0.0.tgz",
+      "resolved": "https://na.artifactory.swg-devops.com/artifactory/api/npm/wcp-alchemy-containers-team-npm-virtual/@aws-crypto/crc32/-/crc32-3.0.0.tgz?dl=https%3A%2F%2Fregistry.npmjs.org%2F%40aws-crypto%2Fcrc32%2F-%2Fcrc32-3.0.0.tgz",
```

Comment:
Need to redo package updates without artifactory proxy.

Reply:
done